### PR TITLE
Feature/enable poaps for other daos

### PR DESCRIPTION
**Pull request by amaredeus**
* remove poap guild restrictions
* remove help guild restrictions
* misc fixes

### CODE REVIEW

File: src/app/commands/poap/POAP.ts [hunk 1]
```diff
@@ -11,19 +11,17 @@ import EarlyTermination from '../../errors/EarlyTermination';
 import EndPOAP from '../../service/poap/EndPOAP';
 import DistributePOAP from '../../service/poap/DistributePOAP';
 import ConfigPOAP from '../../service/poap/ConfigPOAP';
```
Comment (nonsensecodes) [inline src/app/commands/poap/POAP.ts:13]: By removing the `discordServerIds` import, do you have another idea of how to manage a phased rollout?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
I just have one question regarding this PR. Just want to make sure we're setting up for clean phased rollout.

File: src/app/commands/poap/POAP.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/commands/poap/POAP.ts:13]: I remember there was an issue with the commands not getting registered if the `guildIDs` wasn't present. Turns out that the bot has to be manually kicked and then reinvited for the commands to take affect. 

Regarding phased rollout, we'll be needing this and it'll have to include the 10 discord Ids. Let me put this restriction back

Comment (nonsensecodes) [reply src/app/commands/poap/POAP.ts:13]: Sounds good. Thank you, Brian 👍🙏

**Review by nonsensecodes — APPROVED** [review]
Looks good! Thank you for updating this, @amaredeus. Let's merge this bitch.

**Pull request merged**